packages/components/src/heading/stories/index.story.tsx

```diff
@@ -10,7 +10,8 @@ import { Heading } from '..';
 
 const meta: Meta< typeof Heading > = {
 	component: Heading,
-	title: 'Components (Experimental)/Heading',
+	title: 'Components (Experimental)/Typography/Heading',
```

Comment:
Experimental components will fall in their own section down below, meaning they will have their own Typography directory. Is that desired? Should we consider marking the experimental components just with badges, like `CustomSelectControlV2` is marked WIP?

cc @ciampo @mirka